README.md

```diff
@@ -1,3 +1,21 @@
 # scalajs-java-securerandom
 
 `scalajs-java-securerandom` provides `java.security.SecureRandom` for Scala.js, in browsers and in Node.js.
+
+## Usage
+
+Use the following dependency:
+
+```scala
+libraryDependencies += ("org.scala-js" %%% "scalajs-java-securerandom" % "1.0.0-SNAPSHOT").cross(CrossVersion.for3Use2_13)
+```
+
+When using a `crossProject`, add the above in `.jsSettings(...)`.
+
+You can then use `java.security.SecureRandom` from your code, and by extension, the `java.util.UUID.randomUUID()` method.
+
+When running on environments other than browsers or Node.js, attempts to use the above features will throw a `java.lang.UnsupportedOperationException`.
```

Comment:
This is somewhat pessimistic. Other runtimes such as Cloudflare Workers (serverless) and Deno have also implemented the WebCrypto APIs.
https://developers.cloudflare.com/workers/runtime-apis/web-crypto/#methods
https://deno.com/blog/v1.11#more-web-crypto-apis-supported

Reply:
Is it better now?

Reply:
Yes! Thanks :)